trade-finance-manager-ui/server/controllers/case/index.ff-test.js

```diff
@@ -0,0 +1,121 @@
+import { AMENDMENT_STATUS, isTfmFacilityEndDateFeatureFlagEnabled } from '@ukef/dtfs2-common';
+import caseController from '.';
+import api from '../../api';
+import { mockRes } from '../../test-mocks';
+
+jest.mock('@ukef/dtfs2-common', () => ({
+  ...jest.requireActual('@ukef/dtfs2-common'),
+  isTfmFacilityEndDateFeatureFlagEnabled: jest.fn(),
+}));
+
+const res = mockRes();
+
+const TOKEN = 'test-token';
+
+const SESSION = {
```

Comment:
Can we use lowercase consts for `TOKEN` and `SESSION` please? Otherwise this will be the only instance of uppercase token/session consts

Reply:
Good spot, have updated.